plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -862,5 +876,18 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		return builder;
 	}
 
+		<#if data.spawnInRaids>
+		@Override public boolean isAlliedTo(Entity ent) {
+		if (super.isAlliedTo(ent)) {
+			return true;
+		} else if (ent instanceof LivingEntity && ((LivingEntity)ent).getMobType() == MobType.ILLAGER) {
+			return this.getTeam() == null && ent.getTeam() == null;
+		} else {
+			return false;
+			}
+		}
+
+	   	@Override public void applyRaidBuffs(int num, boolean logic) {}

```

Comment:
I don't know what it does exactly, but if you think it could be used for more personalization, there should be a procedure here too (only available if raids are enabled).

Reply:
this method is used like this: at the difficulty level of the world it means that the monster (for example the pillager) if the difficulty level is Easy he has a level 1 crossbow enchantment, if the difficulty level is difficult then he will have the enchantment at level 2 etc.  (I deduced this from the Pillager mob class)

Reply:
Ok, thank you for the info! I'm not sure this is useful for most users, so there is no need for a procedure here. If this is really needed, we can always add the procedure later.

Reply:
Ok